package.json

```diff
@@ -4,10 +4,10 @@
 	"displayName": "Open Policy Agent",
 	"repository": {
 		"type": "git",
-		"url": "https://github.com/tsandall/vscode-opa.git"
+		"url": "https://github.com/open-policy-agent/vscode-opa.git"
 	},
 	"description": "Develop, test, debug, and analyze policies for the Open Policy Agent project.",
-	"version": "0.8.0",
+	"version": "0.10.0",
```

Comment:
0.9.0 wasn't good enough for you? ;)

Reply:
`0.9.0` happens to be the currently published version: https://marketplace.visualstudio.com/items?itemName=tsandall.opa. Looks like that version bump was never pushed.

Reply:
Hoping no other changes are missing also ..